packages/mdc-theme/_functions.scss

```diff
@@ -51,3 +56,66 @@
     @return "light";
   }
 }
+
+// Return the given number if $value >= $min and $value <= $max. If $value falls
+// outside those bounds, return the nearest boundary value ($max or $min).
+@function mdc-theme-clamp-number_($value, $min, $max) {
+  @return max($min, min($max, $value));
+}
+
+// lighten() and darken() require values to be between 0% and 100%.
+@function mdc-theme-clamp-lighten-amount_($percentage) {
```

Comment:
If this is used for both lightening and darkening, then this function name is somewhat confusing.  What about `mdc-theme-clamp-percentage_`?

Also, do you foresee other use cases for `mdc-theme-clamp-number_`, or should we just combine these two into a single function?